cmd/manager/main.go

```diff
@@ -48,6 +48,8 @@ func main() {
 		log.Fatal(err)
 	}
 
+	log.Printf("Setting up Controllers.")
```

Comment:
this does not look related to your change.

Reply:
Right, it was originally added to have more debugging info